src/MOI_wrapper/MOI_wrapper.jl

```diff
@@ -724,6 +724,14 @@ function MOI.get(model::Optimizer, attr::MOI.ObjectiveValue)
     return Cbc_getObjValue(model.inner) + model.objective_constant
 end
 
+function MOI.get(model::Optimizer, ::MOI.RelativeGap)
+    # Get objective value of first available solution.
+    obj_val = MOI.get(model, MOI.ObjectiveValue(1))
+    # Get best objective bound.
+    obj_bnd = MOI.get(model, MOI.ObjectiveBound())
+    return abs((obj_bnd - obj_val) / obj_val)
```

Comment:
x-ref: https://github.com/jump-dev/MathOptInterface.jl/issues/590

What does Cbc report if the objective value is `0.0`?

We should probably implement this as a fallback in MOI, instead of doing it for each solver.